app/config/image_variations.yml

```diff
@@ -20,6 +23,10 @@ ezpublish:
                     reference: ~
                     filters:
                         - { name: geometry/scalewidthdownonly, params: [710] }
+                blog_thumbnail_lightbox:
```

Comment:
This image variation is only used for the blog post line view (when viewing the list of blog posts). So we don't need a lightbox variation.
see template: https://github.com/ezsystems/ezplatform-com/blob/83045e08de7103f21004176340a4d9a2b4076f45/app/Resources/views/line/blog_post.html.twig